storage/innobase/include/srv0srv.h

```diff
@@ -360,8 +360,14 @@ extern bool srv_undo_log_encrypt;
 /** Enable or disable encryption of temporary tablespace.*/
 extern bool srv_tmp_tablespace_encrypt;
 
+enum srv_sys_tablespace_encrypt_enum {
+  SYS_TABLESPACE_ENCRYPT_OFF = 0,
+  SYS_TABLESPACE_ENCRYPT_ON = 1,
```

Comment:
I think it now makes sense to have three different values as "OFF, MASTER_KEY, KEYRING_KEY" @laurynas-biveinis  what do you think?

Reply:
@robgolebiowski call